playground/server/plugins/accept-ch.ts

```diff
@@ -0,0 +1,10 @@
+import type { NitroAppPlugin, RenderResponse } from 'nitropack'
+import { addClientHintResponseHeaders } from '~/utils/client-hints'
+
+export default <NitroAppPlugin> function (nitro) {
```

Comment:
Move this to the middleware, add user agent detection to include headers only for chromium based browsers, add vary on response and include ibfo in route meta.